### ISSUE TITLE
Usage on Windows currently broken

### ISSUE THREAD

**Issue by phil-opp** [phil-opp/cargo-post#14]
See #10. A pull request to fix this escaping problem would be appreciated!

**Comment by fesoliveira014** [comment]
Hey @phil-opp , I finally managed to test `cargo-post` in a fresh project. It seemed to work fine with my modifications though. Could you give me details on the error you got when you ran it?

**Comment by fesoliveira014** [comment]
Here is what I ran:
```
PS C:\Users\fesol\source\repos\cargo-post-test> cargo post build --release
   Compiling cargo-post-test v0.1.0 (C:\Users\fesol\source\repos\cargo-post-test)
    Finished release [optimized] target(s) in 0.37s
Running Post Build Script at C:\Users\fesol\source\repos\cargo-post-test\post_build.rs
    Finished dev [unoptimized + debuginfo] target(s) in 0.01s
     Running `target\post_build_script_manifest\target\debug\post-build-script.exe`
Hello from post build script!
CRATE_BUILD_COMMAND: cargo build --release
CRATE_MANIFEST_DIR: C:\Users\fesol\source\repos\cargo-post-test
CRATE_MANIFEST_PATH: C:\Users\fesol\source\repos\cargo-post-test\Cargo.toml
CRATE_OUT_DIR: C:\Users\fesol\source\repos\cargo-post-test\target\release
CRATE_PROFILE: release
CRATE_TARGET:
CRATE_TARGET_DIR: C:\Users\fesol\source\repos\cargo-post-test\target
CRATE_TARGET_TRIPLE:
```

I used the example provided in the readme for the post build script:

```rust
use std::env;

fn main() {
    println!("Hello from post build script!");
    for (key, value) in env::vars() {
        if key.starts_with("CRATE_") {
            println!("{}: {}", key, value);
        }
    }
}
```

**Comment by phil-opp** [comment]
The CI fails when running `cargo post build --package simple` in the https://github.com/phil-opp/cargo-post/tree/master/tests/simple folder. See https://github.com/phil-opp/cargo-post/runs/887228194 for the CI logs.

**Comment by fesoliveira014** [comment]
I got a different error when running the tests locally with my patch:

```
PS C:\Users\fesol\source\repos\cargo-post\tests\simple> cargo post build --package simple
    Finished dev [unoptimized + debuginfo] target(s) in 0.01s
Running Post Build Script at C:\Users\fesol\source\repos\cargo-post\tests\simple\post_build.rs
calling cargo for post build...
   Compiling post-build-script v0.1.0 (C:\Users\fesol\source\repos\cargo-post\tests\target\post_build_script_manifest)
error: expected item, found `[`
 --> Cargo.toml:1:1
  |
1 | [package]
  | ^ expected item

error: aborting due to previous error

error: could not compile `post-build-script`.

To learn more, run the command again with --verbose.
```
As you can see, I print `calling cargo for post build...` right before calling the command in `src/main.rs#242`. It seems like it gets past the parsing of the `Cargo.toml`, but cargo itself doesn't seem to like the manifest. The interesting is that it worked in my fresh project without issue, so the problem might be related to how tests are ran. I will do some further testing on this and see if I can find the cause.

**Comment by mutantcows** [comment]
I've submitted a pull request that addresses this issue. @phil-opp any chance you could merge?

**Comment by phil-opp** [comment]
Should be fixed by #15.
